using-the-website/bonds/bond_ohm-weth.md

```diff
@@ -0,0 +1,21 @@
+# OHM-WETH LP Bond
+
+OHM-WETH LP bond was introduced on 26th October 2021. It allows you to trade [OHM-WETH LP token](https://app.sushi.com/add/0x383518188c0c6d7730d91b2c03a03c837814a899/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2) in exchange for discounted OHM on Olympus.
```

Comment:
The link is actually for adding LP and not swapping as described.

Reply:
I think this is fine. It is not intended for user to swap LP tokens for OHM, but telling them how to get those LP tokens (by adding liquidity) in order to purchase bonds.

Reply:
Should I change it or squash it? :)

Reply:
I think it's good and just approved it. You can merge it